### ISSUE TITLE
multi-variable logistic regression 

### ISSUE THREAD

**Issue by klmartinez** [statsmodels/statsmodels#8981]
Does the sm.Logit() function automatically perform multi-variable/multiple logistic regression so that we can incorporate multiple covariates/predictors and the estimated coefficients are adjusted for the other covariates/predictors? 

For example, this function was used to do a logistic regression on dichotomous disease outcome (case vs control) ~ genotype  @ SNP_A + age, however, when we tested this on a different genotype [dichotomous disease outcome (case vs control) ~ genotype  @ SNP_B + age], the coefficient remained exactly the same for age.

**Comment by Aniket99coder** [comment]
Yes, the sm.Logit() function from the statsmodels library in Python can be used to perform multi-variable or multiple logistic regression. When you include multiple covariates/predictors, the coefficients are adjusted for the presence of the other covariates.
